### PR TITLE
Add mouse events support on x11

### PR DESCRIPTION
**Pull request by HidenoriMatsubayashi**
I added mouse events support on x11.

## Related issues
- https://github.com/sony/flutter-embedded-linux/issues/37

### CODE REVIEW

**Comment by HidenoriMatsubayashi** [comment]
@Mai-Matsuura Please review this PR.

File: src/flutter/shell/platform/linux_embedded/window/linuxes_window_x11.cc [hunk 1]
```diff
@@ -137,4 +172,36 @@ void LinuxesWindowX11::SetClipboardData(const std::string& data) {
...
+        flutter_button = kFlutterPointerButtonMouseMiddle;
+        break;
+      case kButtonBack:
+        flutter_button = kFlutterPointerButtonMouseBack;
+        break;
+      case kButtonForward:
+        flutter_button = kFlutterPointerButtonMouseForward;
+        break;
+      default:
+        break;
```
Comment (Mai-Matsuura) [inline src/flutter/shell/platform/linux_embedded/window/linuxes_window_x11.cc:197]: In the default case, I think it is better to output the error log and return.

**Review by Mai-Matsuura — COMMENTED** [review]
Please see my comment.

File: src/flutter/shell/platform/linux_embedded/window/linuxes_window_x11.cc (hunk 1 above)
Comment (HidenoriMatsubayashi) [reply src/flutter/shell/platform/linux_embedded/window/linuxes_window_x11.cc]: I've added the error handling.

**Comment by Mai-Matsuura** [comment]
@HidenoriMatsubayashi I checked the changes and it looks fine.

**Pull request merged**